packages/rocketchat-livechat/client/views/app/livechatAppearance.js

```diff
@@ -161,7 +182,7 @@ Template.livechatAppearance.onCreated(function() {
 	this.offlineSuccessMessage = new ReactiveVar(null);
 	this.titleOffline = new ReactiveVar(null);
 	this.colorOffline = new ReactiveVar(null);
-	this.offlineEmail = new ReactiveVar(null);
+	this.emailOffline = new ReactiveVar(null);
```

Comment:
Curious why this change? As `offlineEmail` means that the email to use for offline while `emailOffline` sounds like it should be a boolean or something stating of whether to email them offline or not..

Reply:
This seemed to fix a console error log that I don't remember on top of my head. Happened when the content of the offline email field changed.

You have a point about semantics. it sounds like a boolean, but all other variables are this way as well. `titleOffline`, `colorOffline`...